crates/sui-rpc-benchmark/src/lib.rs

```diff
@@ -54,6 +55,8 @@ pub enum Command {
         duration_secs: u64,
         #[clap(long, default_value = "requests.jsonl")]
         requests_file: String,
+        #[clap(long, value_delimiter = ',', value_parser = value_parser!(String))]
+        methods_to_skip: HashSet<String>,
```

Comment:
leaving `methods_to_skip` as CLI arg list with delimited by `,` instead of a file for simplicity